docs/site/Understanding-the-differences.md

```diff
@@ -61,28 +61,229 @@ it is independent of the three services. We can define the APIs in facade the
-| Tooling              | loopback-cli and API Connect UI                                                                                                                      | [@loopback/cli](https://github.com/strongloop/loopback-next/tree/master/packages/cli)                                                                                                                       |
+<table>
+<thead>
+<tr>
+  <th>Concept/Feature</th>
+  <th>LoopBack 3.x</th>
+  <th>LoopBack 4</th>
+</tr>
+</thead>
+<tbody>
+
+<tr>
+  <td>Programming Language</td>
+  <td>Built with JavaScript ES5<br>Node.js callbacks</td>
+  <td>Modern TypeScript with latest JavaScript features
+    <br>Promises & async/await, ES2016/2017 and beyond
+  </td>
+</tr>
+
+<tr>
+  <td>Core foundation</td>
+  <td>Express with LoopBack extensions</td>
+  <td>Home-grown IoC container</td>
+</tr>
+
+<tr>
+  <td>Tooling</td>
+  <td><code>loopback-cli</code> and API Connect UI</td>
+  <td><a href="Command-line-interface.html"><code>@loopback/cli</code></a></td>
+</tr>
+
+<tr>
+  <td>Model Definition</td>
+  <td>Models can be defined with JavaScript or JSON</td>
+  <td>Models can be defined with TypeScript;
+    <br>TBD: JavaScript - see
+    <a href="https://github.com/strongloop/loopback-next/issues/560">loopback-next#560</a>
+  </td>
+</tr>
+
+<tr>
+  <td>Model Persistence</td>
+  <td>A model can be attached to a datasource backed by a connector that
+    implements CRUD operations
+  </td>
+  <td><a href="Repositories.html">Repositories</a> are introduced to represent
+    persistence related operations; a repository binds a model metadata
+    to a datasource
+  </td>
+</tr>
+
+<tr>
+  <td>Model Relation</td>
+  <td>Relations can be defined between models</td>
+  <td>Relations can be defined between models; queries and persistence are
+    implemented at repository level
+  </td>
+</tr>
+
+<tr>
+  <td>Model Remoting</td>
+  <td>JavaScript/JSON remoting metadata is used to describe method signatures
+    and their mapping to REST/HTTP
+    <br>Swagger specs are generated after the fact
+  </td>
+  <td>Remoting metadata can be supplied by OpenAPI JSON/YAML documents or
+    generated automatically through TypeScript decorators
+  </td>
+</tr>
+
+<tr>
+  <td>API Spec</td>
+  <td>Swagger 2.0</td>
+  <td>OpenAPI Spec v3 and potentially other API specs such as GraphQL, gRPC, etc.</td>
+</tr>
+
+<tr>
+  <td>API Explorer</td>
+  <td>Built-in UI based on swagger-ui v2 (<code>/explorer</code>)</td>
+  <td>Built-in UI based on swagger-ui v3 (<code>/explorer</code>)</td>
+</tr>
+
+<tr>
+  <td>DataSource</td>
+  <td>JSON</td>
+  <td>TypeScript and JSON;</td>
```

Comment:
`<td>TypeScript and JSON;</td>` -> `<td>TypeScript and JSON;`

Reply:
Good catch 👍